src/api/app/controllers/webui/package_controller.rb

```diff
@@ -849,6 +881,19 @@ def package_params
     params.require(:package).permit(:name, :title, :description)
   end
 
+  def package_details_params
```

Comment:
This is a code smell to me, having a second strong parameters method. We should only have `package_params`.

Reply:
I've explained why I opted to have a second parameter method below. I know its a smell, but if I touch this now, this PR will go wildly out of scope.

Reply:
I can refactor this in another PR

Reply:
Sure